examples/gno.land/r/demo/authbanker/authbanker.gno

```diff
@@ -0,0 +1,78 @@
+package authbanker
+
+import (
+	"errors"
+	"std"
+	"strings"
+
+	"gno.land/p/demo/auth"
+	"gno.land/p/demo/ufmt"
+	"gno.land/r/demo/authreg"
+)
+
+// This example is there mostly to demonstrate auth usage, it is quite limited
+// only EOAs can fund accounts
+
+const denom = "ugnot"
+
+var vaults = make(map[string]int64)
+
+// GetCoins returns the amount of coins in the vault identified by `entityID`
+func GetCoins(entityID string) int64 {
+	return vaults[entityID]
+}
+
+// SendCoins sends `amount` coins from the vault identified by `atok` to the account identified by `to`.
+//
+// `to` can be an entity ID or an address.
+func SendCoins(atok auth.Token, to string, amount int64) {
```

Comment:
Can you write a txtar or share a few `maketx` calls that you expect to use with this?

Reply:
There is an integration test [here](https://github.com/gnolang/gno/pull/3406/files#diff-cbe9b89ff80ea0da20527aeb5fd63ff4183ce61468fb34178653d50688cdf0c4R13)

Reply:
Added txtar [here](https://github.com/gnolang/gno/pull/3406/commits/024c5e242200268cc58fdc2fd4dc5fb13b69c1a5)